### ISSUE TITLE
Error when clicking non-translatable files field in 2. language

### ISSUE THREAD

**Issue by moritzebeling** [getkirby/kirby#3220]
**Describe the bug**  
Non-translatable `files` field throws an error when clicking it in the secondary language.

**To Reproduce**  
1. Have a multilang Kirby setup
2. Create an image field, that is non-translatable
```yaml
coverImage:
  type: files
  multiple: false
  translate: false
```
3. Upload an image to the Kirby panel
4. Switch to the default language and try to select an image. It works, great! Clear the field again.
5. Switch to the secondary language. The "Select"-button is now hidden
6. Klick on the field anyways. The following error-message appears:

```
undefined is not an object (evaluating 'this.$refs.picker.toggle')
```

**Screenshots**  
<img width="370" alt="Bildschirmfoto 2021-03-22 um 23 56 27" src="https://user-images.githubusercontent.com/39742729/112068504-568ff000-8b6a-11eb-91bf-ae22c53e1c7b.png">
<img width="362" alt="Bildschirmfoto 2021-03-22 um 23 56 31" src="https://user-images.githubusercontent.com/39742729/112068513-5859b380-8b6a-11eb-8579-e67077a572e5.png">

**Kirby Version**  
3.5.3.1

**Machine:**  
Safari on Mac

**Comment by bastianallgeier** [comment]
✅

**Comment by AnnikaGruca-JUNO** [comment]
Using the same file settings as above, the picker-toggle triggers a similar error message for my non-translatable fields (Kirby version 3.5.4).

![kirby-image-bug](https://user-images.githubusercontent.com/77665371/116524696-f82bff00-a8d7-11eb-86f7-02bccf34777f.png)

**Comment by aemkei** [comment]
Alles klar. Dann wissen wir erstmal Bescheid und warten auf ein Update von Kirby.
Danke!

**Comment by afbora** [comment]
@AnnikaVennemann-JUNO This issue fixed on `3.5.5` and `3.5.5` RC will be released on May 4th.
I've tested again on develop branch to sure and works great.
